### PR TITLE
Add code samples to Web Requests

### PR DESCRIPTION
**Pull request by malandrina**

### CODE REVIEW

File: book/strategies/web_requests.md [hunk 1]
```diff
@@ -3,15 +3,38 @@
...
 Many applications ask users to provide their current location in order to
-perform a search for addresses nearby. Often, it's possible to automatically
-retrieve this information from the browser request.
+perform a search for addresses nearby. Often, it's possible to retrieve this
+information from the browser request.
 
 ### Calculating Coordinates
 
-[Geocoder]((https://github.com/alexreisner/geocoder#request-geocoding-by-ip-address)
+[Geocoder](https://github.com/alexreisner/geocoder#request-geocoding-by-ip-address)
```
Comment (joshuaclayton) [inline book/strategies/web_requests.md:11]: :thumbsup:

File: book/strategies/web_requests.md [hunk 2]
```diff
@@ -3,15 +3,38 @@
...
+       "longitude"=>-122.2728,
+       "metro_code"=>"807",
+       "areacode"=>"510"
+     }>
+```
+
 On the client side, the [W3C Geolocation API](http://dev.w3.org/geo/api/spec-source.html)
-provides location data that can be access [via
-JavaScript](http://diveintohtml5.info/geolocation.html#the-code).
+provides location data that can be accessed [with
```
Comment (joshuaclayton) [inline book/strategies/web_requests.md]: "that can be accessed" => "accessible"?

File: book/strategies/web_requests.md [hunk 3]
```diff
@@ -3,15 +3,38 @@
...
+JavaScript](http://diveintohtml5.info/geolocation.html#the-code):
+
+```javascript
+navigator.geolocation.getCurrentPosition(successCallback, failureCallback)
```
Comment (joshuaclayton) [inline book/strategies/web_requests.md]: This'll probably need to end with a semicolon

File: book/strategies/web_requests.md (hunk 2 above)
Comment (malandrina) [inline book/strategies/web_requests.md]: Yes. Done.

**Pull request merged**